### DISCUSSION TITLE
Java - creating a new thread

### DISCUSSION THREAD

**Discussion by barovinh** [caro-game-but-pay-to-win-java/caro-game-but-pay-to-win-java#2] [Q&A] [answered]
Thread t1 = new Thread(new Runnable() {
    public void run()
    {
         // code goes here.
    }});
    t1.start();

**Comment by vthanhduong** [comment, marked as answer]
new Thread(() ->{
        System.out.println("Does it work?");
        Thread.sleep(1000);
        System.out.println("Nope, it doesnt...again.");       
}){{start();}}.join();
